src/classes/os-utils.ts

```diff
@@ -57,6 +58,21 @@ export default class OsUtils {
     }
   }
 
+  public static detectOS = () : string => {
+    let detectedOS : string;
+    if(windows){
+      detectedOS = "WINDOWS";
+    }
+    if(linux){
```

Comment:
Add space between if-blocks
```suggestion

    if(linux){
```